io_directory_example/settings/lens_system3_config.yml

```diff
@@ -41,14 +57,14 @@ kwargs_constraints:
 fitting:
   pso: true
   pso_settings:
-    num_particle: 10
-    num_iteration: 10
+    num_particle: 2
```

Comment:
Why do you need to change these PSO settings? Is this config file being used to run a model fitting?

Reply:
In additional to running pytest, I usually like to run the lens_system3 model to make sure everything is running properly

Reply:
Alright, you can actually make a new test function to do that run instead of from a notebook. There's nothing bad about more test functions.